static/app/components/modals/recoveryOptionsModal.spec.tsx

```diff
@@ -101,4 +107,32 @@ describe('RecoveryOptionsModal', function () {
       screen.queryByRole('button', {name: 'Add a Phone Number'})
     ).not.toBeInTheDocument();
   });
+
+  it('renders the LoadingIndicator while fetching authenticators', async () => {
+    MockApiClient.clearMockResponses();
+    MockApiClient.addMockResponse({
+      url: '/users/me/authenticators/',
+      method: 'GET',
+      body: [],
+    });
+
+    renderComponent();
+
+    const indicator = await screen.findByTestId('loading-indicator');
+    expect(indicator).toBeInTheDocument();
+  });
```

Comment:
these tests are often race conditions between checking for the loading state and the request being finished. Would recommend not including it.